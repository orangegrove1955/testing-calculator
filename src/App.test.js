import React from "react";
import App from "./App";
import { shallow } from "enzyme";

describe("App component", () => {
  it("should render a <div />", () => {
    const wrapper = shallow(<App />);
    expect(wrapper.find("div").length).toEqual(1);
  });
});
