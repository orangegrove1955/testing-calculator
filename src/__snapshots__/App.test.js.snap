// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should match App snapshot render 1`] = `ShallowWrapper {}`;
